### ISSUE TITLE
Training data for EfficientViT-L2

### ISSUE THREAD

**Issue by P3ngLiu** [mit-han-lab/efficientvit#18]
Hi, I 'm just curious about the training data of those pretrained EfficientViT-L2. Is it trained on ImageNet-1K or ImageNet-22K?

**Comment by han-cai** [comment]
They are trained on ImageNet-1K with random initialization for 300 epochs + 20 warmup epochs using supervised learning.
